quadkey/bq/test/quadkey_integration.js

```diff
@@ -0,0 +1,171 @@
+const assert = require('assert').strict;
+const {BigQuery} = require('@google-cloud/bigquery');
+
+const BQ_PROJECTID = process.env.BQ_PROJECTID;
+const BQ_DATASET_QUADKEY = process.env.BQ_DATASET_QUADKEY;
+
+describe('QUADKEY integration tests', () => {
+
+    let client;
+    before(async () => {
+        if (!BQ_PROJECTID) {
+            throw "Missing BQ_PROJECTID env variable";
+        }
+        if (!BQ_DATASET_QUADKEY) {
+            throw "Missing BQ_DATASET_QUADKEY env variable";
+        }
+        client = new BigQuery({projectId: `${BQ_PROJECTID}`});
+    });
+  
+    it ('Returns the proper version', async () => {
+        const query = `SELECT \`${BQ_PROJECTID}\`.\`${BQ_DATASET_QUADKEY}\`.VERSION() as versioncol;`;
+        let rows;
+        await assert.doesNotReject( async () => {
+            const [job] = await client.createQueryJob({ query: query });
+            [rows] = await job.getQueryResults();
+        });
+        assert.equal(rows.length, 1);
+        assert.equal(rows[0].versioncol, 1);
+    });
+
+    it ('Should be able to encode/decode quadints at different zooms', async () => {
+        let tilesPerLevel, x, y;
+        for(let z = 1; z < 30; z = z + 7)
```

Comment:
Please don't do a single query per zoom level, this is going to be awfully slow. Instead compose a query with all the things you want to check and send a single query to BigQuery and then check the results. You should also add tiles in between the first and last, and check what happens when invalid values are passed (if we want to handle it).

Same for ALL other integration tests.